doc/content/source/kernels/AxisymmetricCurlZ.md

```diff
@@ -1,20 +1,25 @@
 # AxisymmetricCurlZ
 
-!alert construction title=Undocumented Class
-The AxisymmetricCurlZ has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /Kernels/AxisymmetricCurlZ
 
 ## Overview
 
-!! Replace these lines with information regarding the AxisymmetricCurlZ object.
+`AxisymmetricCurlZ` computes that the Z-component of an axisymmetric curl.
+
+The Z-component of an axisymmetric curl is defined as:
+
+\begin{equation}
+\left( \nabla \times \bf{A} \right)_z = \frac{\text{A}_\phi}{r} + \frac{\partial \text{A}_\phi}{\partial r}
+\end{equation}
+
+where $\left( \nabla \times \bf{A} \right)_z$ is the Z-component of the curl of vector $\bf{A}$, $\text{A}_\phi$ is the azimuth component of vector $\bf{A}$, and $r$ is the radial coordinate.
```

Comment:
`r` in this case is not denoting a component of a vector, but is the radial position of the calculation (for FEM, this would be the radial  coordinate of the current quadrature point). I can switch the wording to `radial position`, if that clarifies it better.

Reply:
I think the radial position language would be great here!